### PR TITLE
Use test.yml for the badge as we have no CI workflow

### PR DESCRIPTION
**Pull request by Jamstah**

### CODE REVIEW

File: cover.out [hunk 1]
```diff
@@ -0,0 +1,176 @@
+mode: set
```
Comment (milosgajdos) [inline cover.out:1]: This file should be `gitignore`-d 🤔

Comment (thaJeztah) [reply cover.out:1]: It will be, in the other PR.

`git add -u` can be your friend as well 😂

Comment (Jamstah) [reply cover.out]: :flushed:

**Comment by thaJeztah** [comment]
All looking green with this PR!

...multiple shades of it, or is it just me that the "test" and "license scan" badges have a different shade of green?

Screenshot 2023-08-31 at 12 24 18

**Review by thaJeztah — APPROVED** [review]
LGTM

**Review by milosgajdos — APPROVED** [review]

**Pull request merged**